docs/features/traffic-management/nginx.md

```diff
@@ -24,12 +26,15 @@ spec:
       stableService: stable-service  # required
       trafficRouting:
         nginx:
-           primaryIngress: primary-ingress  # required
-           annotationPrefix: example.nginx.com/ # optional
+           stableIngress: primary-ingress  # required
+           annotationPrefix: customingress.nginx.ingress.kubernetes.io # optional
+           additionalIngressAnnotations: #optional
+             canary-by-header: X-Canary
+             canary-by-header-value: iwantsit
```

Comment:
Might be nice to link to https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#canary, when I got here I was thinking "where did these come from?"

Reply:
That should already be there (first paragraph).

Reply:
oops 👍